client/src/components/campaign/round-new/round-new.html

```diff
@@ -0,0 +1,165 @@
+<div class="round round--edit round--last"
+      layout="row" layout-align="center start">
+  <div class="round__status"
+        layout="row" layout-align="center center"
+        ng-if="$ctrl.showForm">
+    <span ng-if="$ctrl.round.vote_method === 'yesno'"><md-icon>thumbs_up_down</md-icon></span>
+    <span ng-if="$ctrl.round.vote_method === 'rating'"><md-icon>star_border</md-icon></span>
+    <span ng-if="$ctrl.round.vote_method === 'ranking'"><md-icon>sort</md-icon></span>
+  </div>
+  <span class="round__line"></span>
+  <md-button ng-if="!$ctrl.showForm"
+    ng-click="$ctrl.showForm = true">
+    <md-icon>add</md-icon>
+    Add Round
+  </md-button>
+  <span ng-if="!$ctrl.showForm" flex></span>
+  <form role="form" name="$ctrl.roundEditForm" flex ng-if="$ctrl.showForm">
+    <fieldset ng-disabled="$ctrl.loading">
+    <h2 class="round__name"
+        layout="row" layout-align="space-between center">
+      <div layout="column" layout-align="center start">
+        <span>{{ $ctrl.round.name }}</span>
+        <span class="muted">{{ "round." + $ctrl.round.vote_method | translate }}</span>
+      </div>
+    </h2>
+    <md-card class="round__description" flex>
+      <md-card-content><fieldset>
+        <div layout="row" layout-align="center stretch">
+          <div class="round__directions" flex="60"
+              layout="column" layout-align="start stretch">
+
+            <md-input-container class="no-margin-bottom">
+              <label>Round Name</label>
+              <input type="text" autofocus
+                  ng-model="$ctrl.round.name"
+                  ng-required="true">
+            </md-input-container>
+
+            <div layout="row" layout-align="start start">
+              <div flex="60">
+                <h4>Deadline Date</h4>
+                <div layout="row" layout-align="start center">
+                  <md-datepicker ng-model="$ctrl.round.deadline_date"
+                      ng-required="true"
```

Comment:
As a design note, it would be good to have an icon for required items. It's easy to miss that this date is mandatory!